contrib/ansible/roles/docker/handlers/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: restart docker
+  command: systemctl daemon-reload
+  notify:
+    - restart dockerservice
+
+- name: restart dockerservice
+  service: name=docker state=restarted
```

Comment:
hmmm, would different naming make more sense?  maybe reload systemd and restart docker, and have the callers notify both?